### PR TITLE
feat: Update 2022 year in review financials

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

Updated the 2022 year in review post.

#### What changes did you make? (Give an overview)

Updated the financials to match the way we calculated them for 2023 and added the list of top sponsors.

### CODE REVIEW

**Comment by mdjermanovic** [comment]
This expense isn't included:

https://opencollective.com/eslint/expenses/89866

File: src/content/blog/2023-01-10-eslint-2022-year-review.md [hunk 1]
```diff
@@ -57,37 +57,74 @@ In addition to our normal maintenance and development, we also kicked off a coup
...
+## Top sponsors
+
+The overwhelming majority of our income came from sponsors, and we're extra grateful to all of the sponsors who donated $1,000 or more during the year.
+
+| **Sponsor** | **2022 Donation Total** |
+|-------------|-------------------------|
+| Chrome Frameworks Fund | $23,500.00 |
+| Automattic | $22,000.00 |
+| Etsy Open Source | $18,000.00 |
+| BuySellAds | $13,831.16 |
```
Comment (mdjermanovic) [inline src/content/blog/2023-01-10-eslint-2022-year-review.md:84]: Ths is not a sponsor.

**Comment by nzakas** [comment]
Added it to the tech writing interviews line item.

File: src/content/blog/2023-01-10-eslint-2022-year-review.md [hunk 2]
```diff
@@ -57,37 +57,74 @@ In addition to our normal maintenance and development, we also kicked off a coup
...
+| Icons8 | $2,200.00 |
+| Discord | $2,200.00 |
+| Anagram Solver | $2,200.00 |
+| ThemeIsle | $2,000.00 |
+| RIDI | $2,000.00 |
+| VPS Server | $1,800.00 |
+| Sentry | $1,608.00 |
+| HeroCoders | $1,400.00 |
+| QuickBooks Tool hub | $1,320.00 |
+| Yannick Croissant | $1,200.00 |
```
[the remark this answers is not in the document]
Comment (mdjermanovic) [reply src/content/blog/2023-01-10-eslint-2022-year-review.md]: I'm not sure why exactly, but it looks like some transactions are not included in the sum. For example, Automattic: I clicked on its amount, and another tab was created ("Detail1-Automattic", I believe you can see it as well right after "Open Collective Sponsors Total"). It has 11 transactions. The one that seems to be not included is from 2022-01-05 (shortId 247cbe89).

[the remark this answers is not in the document]
Comment (mdjermanovic) [reply src/content/blog/2023-01-10-eslint-2022-year-review.md]: It could be because of the range:

image

[the remark this answers is not in the document]
Comment (nzakas) [reply src/content/blog/2023-01-10-eslint-2022-year-review.md]: That would do it! Thanks.

File: src/content/blog/2023-01-10-eslint-2022-year-review.md [hunk 3]
```diff
@@ -113,3 +150,5 @@ In 2022, the ESLint project donated $14,350.00 USD to ecosystem projects and our
 While ESLint had a fantastic 2022, we are looking forward to an even better 2023. With the ESLint Community and documentation projects running, we expect to make the ESLint ecosystem even stronger. We are also planning to start on a [complete rewrite](https://github.com/eslint/eslint/discussions/16557) from the ground up to make sure that ESLint is ready for the next ten years of JavaScript development. This process will likely take place in small, incremental steps as we think through how JavaScript development has changed and will continue to evolve in the future.
 
 We are excited about the future of ESLint and we appreciate your ongoing support.
+
+**Updated (2024-02-22):** We revised how our financials were reported to correct for errors in our initial calculation and to bring the numbers in this post inline with our 2023 approach to calculating our financials.
```
Comment (mdjermanovic) [inline src/content/blog/2023-01-10-eslint-2022-year-review.md:154]:
```suggestion
**Updated (2024-02-29):** We revised how our financials were reported to correct for errors in our initial calculation and to bring the numbers in this post inline with our 2023 approach to calculating our financials.
```

**Review by mdjermanovic — APPROVED** [review]
LGTM, I just left one number formatting suggestions, and we might want to update the date of this change.

**Pull request merged**